### ISSUE TITLE
What's the purpose of "ignoring last label" in this loss function?

### ISSUE THREAD

**Issue by lishen** [aurora95/Keras-FCN#66]
See code:

https://github.com/aurora95/Keras-FCN/blob/35afe12e514a3bf5e56bc90e69d5b329ce9ae68d/utils/loss_function.py#L14
https://github.com/aurora95/Keras-FCN/blob/35afe12e514a3bf5e56bc90e69d5b329ce9ae68d/utils/loss_function.py#L15
https://github.com/aurora95/Keras-FCN/blob/35afe12e514a3bf5e56bc90e69d5b329ce9ae68d/utils/loss_function.py#L16

At first you appended an additional column of all 0's in one-hot coding and then removed it. It seems to me you only need a single call of `K.one_hot` to replace L14-16. Or, did I miss something?

**Comment by Nandayang** [comment]
Hi, I have the same question as yours. Do you have any answer about this question?3KS!@lishen
